tidy3d/components/scene.py

```diff
@@ -769,6 +772,10 @@ def plot_structures_eps(
             The x range if plotting on xy or xz planes, y range if plotting on yz plane.
         vlim : Tuple[float, float] = None
             The z range if plotting on xz or yz planes, y plane if plotting on xy plane.
+        component : str = None
+            Material polarization component to plot for anisotropic materials,
+            e.g. "xx", "yy", "zz", "xy", "yz", ...
```

Comment:
could you put the string in double backslash quotes, eg ``"xx"``? It will render better for the docs. see https://www.notion.so/flexcompute/Python-Style-Guide-afac5dbbf949463e89c18fe39b7453c9?pvs=4#72d1e0acdd8b42b6872ba944a4599efd

note, check this for all other related docstrings too.